### PR TITLE
MotivationES-Update

### PR DESCRIPTION
**Pull request by RafaelDavisH**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

### CODE REVIEW

**Review by naomi-lgbt — APPROVED** [review]

File: client/i18n/locales/espanol/motivation.json [hunk 1]
```diff
@@ -115,6 +115,90 @@
...
+      "author": "Pablo Picasso"
+    },
+    {
+      "quote": "La perfección no es alcanzable pero si la perseguimmos podemos conseguir la excelencia",
+      "author": "Vince Lombardi"
+    },
+    {
+      "quote": "La excelencia no es un acto, es un hábito",
+      "author": "Aristóteles"
+    },
```
Comment (nhcarrigan) [inline client/i18n/locales/espanol/motivation.json:202]:
```suggestion
    }
```
Remove trailing comma to fix linter error.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: client/i18n/locales/espanol/motivation.json (hunk 1 above)
Comment (RafaelDavisH) [reply client/i18n/locales/espanol/motivation.json:202]: Thanks

**Review by RafaelDavisH — COMMENTED** [review]
LGTM

File: client/i18n/locales/espanol/motivation.json (hunk 1 above)
Comment (RandellDawson) [reply client/i18n/locales/espanol/motivation.json:202]: @RafaelDavisH You need to commit @nhcarrigan's suggestion.  You have to click "Commit suggestion" to resolve it.

**Review by RandellDawson — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]

**Comment by dustpan28645** [comment]
Do y'all have a place for this camp and how much it total cost

**Pull request merged**